### ISSUE TITLE
The Port Itself: Combine all three games into one single launcher?

### ISSUE THREAD

**Issue by Shockwave508** [daniele-rapagnani/alephone-android#38]
Instead of separating the three games into their own separate apps, would it be possible to make an "all-in-one" APK that contains the entire trilogy?

**Comment by daniele-rapagnani** [comment]
Yes I think you are right, having a single Aleph One app for all games is probably the best thing. I'm working on an experimental launcher that will be able to download/install the original trilogy as well as scenarios and run them from the same APK.

**Comment by Shockwave508** [comment]
You could probably get in touch with EmileB, from OpenTouch Gaming; he's already specialized in bringing Doom and Quake sourceports to Android (through Delta Touch and Quad Touch, respectively).

**Comment by daniele-rapagnani** [comment]
Yes I'm aware of OpenTouch Gaming ports, they are great but their work is for-profit and I'd really like to keep this port free and open. Unless OTG is willing to donate their code to the project (assuming that it's something that can be integrated in AlephOne) I think contacting them wouldn't be of much help here.

**Comment by Shockwave508** [comment]
You could probably make one single launcher with just the required Aleph One binaries, and allow the user to select the filepaths on his device storage; would remove the need for the AlephOneScenario.txt file, at any rate.

**Comment by daniele-rapagnani** [comment]
Yes exactly, that's what I'm doing with the only exception that unfortunately you can't just point to a folder on your phone and run the game from there. The scenario must be copied/installed to the application's storage directory because Android is changing how applications can access external storage, adopting a mechanism similar to iOS where there's no shared storage at all.

**Comment by Biggusjohnnus** [comment]
Hey, I know this is outta nowhere but could we get a precompiled apk for the latest version of this port? I'd love to finally be able to play M1 with its soundtrack! Unfortunately I am still not exactly smart enough to compile it myself lol.

**Comment by daniele-rapagnani** [comment]
I've released a 0.3.0-alpha, see the release description for the known bugs.
I'll resume working on this one of these days 😃
